TheJumpCloud/support
# fonts via commands example

## Issues
* [SA-2887](https://jumpcloud.atlassian.net/browse/SA-2887) - Script Examples to install fonts via JumpCloud Commands

## What does this solve?

Windows & MacOS Scripts to install fonts on systems via commands

## Is there anything particularly tricky?

## How should this be tested?

Test on windows / macOS vms. Package several open source fonts, zip those fonts & upload to S3, try the script on both a windows and macOS system, the fonts should be installed after the command runs

## Screenshots
